app.rb

```diff
@@ -5,14 +5,17 @@
 require_relative 'student'
 require_relative 'teacher'
 require_relative 'validate_input'
+require_relative 'store_data'
+require_relative 'load_data'
+require 'json'
 
 class App
   attr_accessor :books, :people, :rentals
 
   def initialize
-    @books = []
-    @rentals = []
-    @people = []
+    @books = load_books
+    @rentals = load_rentals
```

Comment:
- [ ] Kindly fix the problem with the `@rentals` list as now the old rentals stored in the JSON file are not loaded when the program started.